src/views/domain-page/domain-page-header-info-item/domain-page-header-info-item.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Skeleton } from 'baseui/skeleton';
+
+import { overrides, styled } from './domain-page-header-info-item.styles';
+import { Props } from './domain-page-header-info-item.types';
+
+export default function DomainPageHeaderInfoItem(props: Props) {
+  return (
+    <styled.ItemContainer>
+      <styled.ItemTitle>{props.title}</styled.ItemTitle>
+      {props.loading ? (
+        <Skeleton
+          width={props.placeholderSize}
+          overrides={overrides.skeleton}
+          animation={true}
+        />
+      ) : typeof props.content === 'string' ? (
```

Comment:
Please avoid nested ternaries for better readability.
We can add the rule in the linting
https://eslint.org/docs/latest/rules/no-nested-ternary

Reply:
Agreed, but I wanted to avoid creating another layer of hierarchy by moving the loaded state to a separate component. But I think I can handle this with styled components directly.